src/styles/Book.css

```diff
@@ -0,0 +1,45 @@
+.book {
+  background-color: #fff;
+  border: 1px solid #e8e8e8;
+  border-radius: 4px;
+  padding: 2rem 1.688rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 1rem 0;
+}
```

Comment:
- [ ] I would suggest you add `comment and edit buttons` to stick with Figma design but you don't need to implement the functionality yet.
### your design
![your](https://user-images.githubusercontent.com/81727988/179338614-196f2519-de09-4bc1-96f8-5b87564f4b9b.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/179338622-4f18cb3e-5efd-41bd-ab6a-051c77b0951f.png)